Gruntfile.js

```diff
@@ -65,10 +65,17 @@ module.exports = function(grunt) {
                     dest:'dist'
                 }
             },
+            admin: {
+                src:'ui/admin.html',
+                nonull: true,
+                options:{
+                    dest:'dist'
+                }
+            },
         },
 
         usemin:{ html:['dist/index.html', 'dist/userguide.html', 'dist/logviewer.html',
-                       'dist/perf.html', 'dist/failureviewer.html'] },
+                       'dist/perf.html', 'dist/failureviewer.html', 'dist/admin.html'] },
```

Comment:
Could you try using something like `['dist/*.html']`? (In a separate, earlier commit)